content/docs/user-guide/visualization/animation/animation-editor/ragdoll/ragdoll-adding-to-animation-graph.md

```diff
@@ -0,0 +1,61 @@
+---
+description: ' Adding a ragdoll to an Animation Graph in the Open 3D Engine Animation Editor. '
+title: Adding a Ragdoll to an Animation Graph
+---
+
+When you create an animation graph to control the ragdoll simulation of your character, you do the following:
++ Prepare the actor asset.
++ Adjust the animation graph to enable the ragdoll.
++ Preview
+
+The animation graph controls the ragdoll simulation of your character. When your character transitions into a blend tree that has a ragdoll node, the ragdoll automatically activates and simulates in game mode in O3DE Editor. When your character transitions out of that state, the ragdoll deactivates. The ragdoll node outputs a bind pose in the Animation Editor.
+
+**To create an animation graph to transition from running state to ragdoll state**
```

Comment:
suggestion: Consider using a heading here (##)

Reply:
I didn't modify the stuff on this page, I just moved it from another page. I'm going to leave this one.